cmake/CMakeLists.txt

```diff
@@ -93,6 +93,7 @@ option(onnxruntime_USE_WINML "Build with WinML support" OFF)
 option(onnxruntime_USE_ACL "Build with ACL support" OFF)
 option(onnxruntime_ENABLE_INSTRUMENT "Enable Instrument with Event Tracing for Windows (ETW)" OFF)
 option(onnxruntime_USE_TELEMETRY "Build with Telemetry" OFF)
+option(onnxruntime_ENABLE_WCOS "Build for Windows Core OS" OFF)
```

Comment:
Nit: this, onnxruntime_USE_WINML, MSVC options and onnxruntime_ENABLE_INSTRUMENT are Windows only options and shouldn't be shown on every platform. Maybe refactor in the future to have them conditionally.

Reply:
good point, but I think this is out of scope for this PR, especially since we a trying to cherry pick this into 1.2 release